config/routes.rb

```diff
@@ -1,6 +1,17 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+  get '/Api/v1/users/:username', to: 'api/v1/users#show'
+  post '/Api/v1/users', to: 'api/v1/users#create'
+  delete '/Api/v1/users/:username', to: 'api/v1/users#destroy'
+
+  get '/Api/v1/cars', to: 'api/v1/cars#index'
+  post '/Api/v1/cars', to: 'api/v1/cars#create'
+  get '/Api/v1/car/:id', to: 'api/v1/cars#show'
+  delete '/Api/v1/cars/:id', to:  'api/v1/cars#destroy'
+
+  get 'Api/v1/users/:user_id/reservations', to: 'api/v1/reservations#index'
+  post 'Api/v1/users/:user_id/reservations', to: 'api/v1/reservations#create'
+  delete 'Api/v1/users/:user_id/reservation/:id', to:  'api/v1/reservations#destroy'
```

Comment:
- [ ]  OPTIONAL: In the industry, we use the #resources method for multiple routes on the same controller.

## Explanation

By using the `only:` keyword parameter, you can generate only the routes that you are using.

[Look](https://launchschool.com/books/demystifying_rails/read/routes_and_resources)

![image](https://github.com/luis-pomare/rails-car-booking-rentals-backend/assets/57421823/0ffc8455-d43c-44c5-a44f-d96bf49e6766)



## This is an example



```ruby
namespace :api do
    namespace :v1 do
      resources :users, only: %i[index update show create destroy]
      resources :reservations, only: %i[index show update create destroy]
      resources :cars, only: %i[index update show create destroy]
    end
  end

```

That way, your routes will be more consistent, and will meet the best industry practices💪